### PR TITLE
added the warning message when a user with bot account gets deleted

### PR DESCRIPTION
**Pull request by pradeepmurugesan**
#### Summary
This adds a disclaimer message regarding the bot accounts when deactivating a user with at-least one bot accounts created.

#### Ticket Link
Fixes https://github.com/mattermost/mattermost-server/issues/10863

#### Related Pull Requests
- Has server changes - No
- Has redux changes - No
- Has mobile changes- No

Screenshots:

Without Bot accounts
With Bot accounts

### CODE REVIEW

**Comment by hanzei** [comment]
@pradeepmurugesan Is this ready for review?

**Comment by jasonblais** [comment]
Thanks @pradeepmurugesan for the update! Looks like the latest build fails - I'll re-test after you've had a chance to take a look at it

**Comment by pradeepmurugesan** [comment]
@jasonblais have modified the same. Kindly review

**Comment by jasonblais** [comment]
@sudheerDev @asaadmahmood Kind reminder to help review this PR when you have a chance

**Comment by asaadmahmood** [comment]
Done @jasonblais. Updated the spacing.
Screenshot 2019-06-09 at 2 19 14 PM

File: components/admin_console/system_users/system_users_dropdown/system_users_dropdown.jsx [hunk 1]
```diff
@@ -134,7 +139,18 @@ export default class SystemUsersDropdown extends React.PureComponent {
...
         e.preventDefault();
-        this.setState({showDeactivateMemberModal: true});
+        if (this.shouldDisableBotsWhenOwnerIsDeactivated()) {
+            this.props.actions.loadBots(
+                Constants.Integrations.START_PAGE_NUM,
+                Constants.Integrations.PAGE_SIZE
+            ).then(
+                (result) => {
+                    if (result.data) {
+                        this.setState({loading: false, showDeactivateMemberModal: true});
```
Comment (sudheerDev) [inline components/admin_console/system_users/system_users_dropdown/system_users_dropdown.jsx]: I think the state variable `loading` is not needed.

minor nit: Most of us are inclined to using `await` nowadays and we could use destructing with it as well making this an obvious pattern you would see around in the codebase.

```
const {data} = await func();
if(data) {
  setState({});
}
```

**Comment by sudheerDev** [comment]
@jasonblais Need another dev reviewer on this.

File: components/admin_console/system_users/system_users_dropdown/system_users_dropdown.jsx [hunk 2]
```diff
@@ -217,13 +217,22 @@ export default class SystemUsersDropdown extends React.PureComponent {
...
+                                    id='deactivate_member_modal.desc.for_users_with_bot_accounts'
+                                    defaultMessage='This action deactivates {username}.'
+                                    values={{
+                                        username: user.username,
+                                    }}
+                                />
+                            </p>
+                            <FormattedMarkdownMessage
+                                id='deactivate_member_modal.desc.for_users_with_bot_accounts_info'
+                                defaultMessage='\n * They will be logged out and not have access to any teams or channels on this system.\n * Bot accounts they manage will be disabled along with their integrations. To enable them again, go to Integrations > Bot Accounts. [Learn more about bot accounts](!https://mattermost.com/pl/default-bot-accounts).'
```
Comment (jasonblais) [inline components/admin_console/system_users/system_users_dropdown/system_users_dropdown.jsx]: @asaadmahmood Is the `\n * ` part intentionally part of the string? Wondering if that should instead handled by the JSX code itself

**Comment by jasonblais** [comment]
@asaadmahmood also, looks like the build fails

File: components/admin_console/system_users/system_users_dropdown/system_users_dropdown.jsx (hunk 2 above)
Comment (asaadmahmood) [reply components/admin_console/system_users/system_users_dropdown/system_users_dropdown.jsx]: Yes, that's intentional. This is a markdown formatted message.

**Comment by pradeepmurugesan** [comment]
@sudheerDev  Fixed the review comments.

@asaadmahmood I am so sorry. I found that I have overwritten your commit during the process of fixing the review comments. Can you kindly push your changes. I am so sorry.

**Comment by asaadmahmood** [comment]
@pradeepmurugesan No worries, will do in a while.

**Comment by asaadmahmood** [comment]
@pradeepmurugesan Done.

**Comment by bradjcoughlin** [comment]
@jasonblais is this PR slated for a release?

**Comment by jasonblais** [comment]
@bradjcoughlin This can go into the 5.14 release (so just merged to master)

@pradeepmurugesan Can you help rebase against the master branch? Looks like this is otherwise ready to merge 🎉

**Comment by hanzei** [comment]
Thanks @pradeepmurugesan for the awesome contribution :+1:

**Pull request merged**